### ISSUE TITLE
Release 6.3.1

### ISSUE THREAD

**Issue by davidschober** [avalonmediasystem/avalon#2716]
## Description

- Redis security fix

**Comment by carrickr** [comment]
Release tagged and some prose put in the draft, ready for PO movement

**Comment by carrickr** [comment]
Actually hold on this, we're having a blacklight issue:

```
tipleIncludedBlocks: Cannot define multiple 'included' blocks for a Concern
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/activesupport-4.2.9/lib/active_support/concern.rb:126:in `included'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/blacklight-6.11.0/app/controllers/concerns/blacklight/catalog.rb:11:in `<module:Catalog>'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/blacklight-6.11.0/app/controllers/concerns/blacklight/catalog.rb:2:in `<top (required)>'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:457:in `load'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:457:in `block in load_file'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:647:in `new_constants_in'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:456:in `load_file'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:354:in `require_or_load'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:317:in `depend_on'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:233:in `require_dependency'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/railties-4.2.9/lib/rails/engine.rb:472:in `block (2 levels) in eager_load!'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/railties-4.2.9/lib/rails/engine.rb:471:in `each'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/railties-4.2.9/lib/rails/engine.rb:471:in `block in eager_load!'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/railties-4.2.9/lib/rails/engine.rb:469:in `each'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/railties-4.2.9/lib/rails/engine.rb:469:in `eager_load!'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/railties-4.2.9/lib/rails/engine.rb:346:in `eager_load!'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/resque-1.27.4/lib/resque/tasks.rb:45:in `each'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/resque-1.27.4/lib/resque/tasks.rb:45:in `block (2 levels) in <top (required)>'
/srv/avalon/avalon_r6/shared/bundle/ruby/2.4.0/gems/rake-12.3.0/exe/rake:27:in `<top (required)>'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/cli/exec.rb:74:in `load'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/cli/exec.rb:74:in `kernel_load'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/cli/exec.rb:27:in `run'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/cli.rb:362:in `exec'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/cli.rb:22:in `dispatch'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/cli.rb:13:in `start'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/exe/bundle:30:in `block in <top (required)>'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/lib/bundler/friendly_errors.rb:121:in `with_friendly_errors'
/home.local/avalon/.rvm/gems/ruby-2.4.1/gems/bundler-1.15.4/exe/bundle:22:in `<top (required)>'
/home.local/avalon/.rvm/gems/ruby-2.4.1/bin/bundle:22:in `load'
/home.local/avalon/.rvm/gems/ruby-2.4.1/bin/bundle:22:in `<main>'
/home.local/avalon/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `eval'
/home.local/avalon/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => resque:work => resque:preload
(See full trace by running task with --trace)```

**Comment by carrickr** [comment]
Not sure why this is happening, there is only one included block in that controller in 6.11

https://github.com/projectblacklight/blacklight/blob/v6.11.0/app/controllers/concerns/blacklight/catalog.rb

**Comment by carrickr** [comment]
This is only occurring on spruce.  6.3.1 has deployed to pawpaw and NU's staging without issue (both AWS running Ruby 2.4) and works fine on my local laptop as well (running 2.3).
